julia/pyjulia_helper.jl

```diff
@@ -1,5 +1,36 @@
 module _PyJuliaHelper
 
+if VERSION < v"0.7-"
```

Comment:
Seems like pyjulia should drop support for pre-1.0 versions of Julia.

Reply:
Yeah, I guess I'll do it after releasing 0.3.  It has no major blocker at the moment so it can happen anytime.  I thought I'd keep Julia 0.6 compatibility to make sure refactoring keeps the flexibility to support separated cache (which in retrospect didn't help much).